cockatrice/src/settingscache.cpp

```diff
@@ -89,6 +91,12 @@ void SettingsCache::setCardInfoViewMode(const int _viewMode) {
     settings->setValue("cards/cardinfoviewmode", cardInfoViewMode);
 }
 
+void SettingsCache::setHighlightWords(const QString &_highlightWords) {
+    // Words are seperated by a comma and you can not use spaces in words
+    highlightWords = _highlightWords.simplified().replace(" ", "").split(",");
```

Comment:
I thought I added a line comment for this already. Its better to pass in the correct type than having the method convert it for you. It causes API confusion. You init `highlightWords` as a list, but you have a set that passed in a string. What if this call is used incorrectly? Try to stick to the original type when possible.